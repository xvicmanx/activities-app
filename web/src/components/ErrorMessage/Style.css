.ErrorMessage {
  text-align: center;
  background-color: #ff3860;
  padding: 10px;
  border-radius: 5px;
  width: fit-content;
  margin: 20px auto;
  color: #fff;
  line-height: 1.6;
  border: 1px solid rgb(230, 59, 94);
  font-weight: 500;
}
