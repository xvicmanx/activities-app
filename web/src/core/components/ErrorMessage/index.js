import React from 'react';
import './Style.css';

const ErrorMessage = ({ text }) => <div className="ErrorMessage">{text}</div>;

export default ErrorMessage;
