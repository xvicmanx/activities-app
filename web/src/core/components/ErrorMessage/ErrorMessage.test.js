import React from 'react';
import renderer from 'react-test-renderer';
import ErrorMessage from './';

describe('ErrorMessage', () => {
  it('renders properly', () => {
    const component = renderer.create(
      <ErrorMessage>Error!</ErrorMessage>,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});