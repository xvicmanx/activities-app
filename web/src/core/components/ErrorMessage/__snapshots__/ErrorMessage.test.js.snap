// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage renders properly 1`] = `
<div
  className="ErrorMessage"
>
  Error!
</div>
`;
